devenv.nix

```diff
@@ -0,0 +1,16 @@
+{ pkgs, lib, ... }:
+
+{
+  # Packages
+  packages = [
+    pkgs.nodePackages.pnpm
```

Comment:
This will use the latest version, is this OK?

Reply:
enable ```
languages.javascript.corepack.enable = true;
```

and remove pnpm as package. so it uses the pinned version in the package.json